packages/react-menu/src/components/Menu/Menu.test.tsx

```diff
@@ -209,7 +215,7 @@ describe('Menu', () => {
     getByRole(role);
   });
 
-  // THID ONE
+  // THIRD ONE
```

Comment:
```suggestion
  // THIRD ONE
```

No idea what I was smoking when I wrote this. Please remove thx

Reply:
remove? your suggestion is the same typo fix as I already did :D